applications/mne_scan/plugins/eegosports/eegosportsdriver.cpp

```diff
@@ -177,6 +243,9 @@ bool EEGoSportsDriver::uninitDevice()
 
     std::cout << "Plugin EEGoSports - INFO - uninitDevice() - Successfully uninitialised the device" << std::endl;
```

Comment:
Please use qInfo(), qDebug() and qWarning() whenever possible. Here I would propose qInfo().